.github/workflows/stale_issue.yml

```diff
@@ -0,0 +1,49 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  push:
+    branches: 
+      - stale_action
+  schedule:
+    - cron: "*/15 * * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v3
+      with:
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: Greetings! Sorry to say but this is a very old issue that is probably not getting as much attention as it deservers. We encourage you to check if this is still an issue in the latest release and if you find that this is still a problem, please feel free to open a new one.
```

Comment:
For this comment. Let's reuse the suggestion I had here: https://github.com/boto/botocore/pull/2028#discussion_r414883535